src/azure-cli/azure/cli/command_modules/iot/_utils.py

```diff
@@ -45,10 +45,7 @@ def open_certificate(certificate_path):
     if certificate_path.endswith('.pem') or certificate_path.endswith('.cer'):
         with open(certificate_path, "rb") as cert_file:
             certificate = cert_file.read()
-            try:
-                certificate = certificate.decode("utf-8")
-            except UnicodeError:
-                certificate = base64.b64encode(certificate).decode("utf-8")
+        certificate = base64.b64encode(certificate).decode("utf-8")
```

Comment:
Just a note to double-check we can cover at least the same cert inputs as before.